### PR TITLE
Adds my Side Project blog post

### PR DESCRIPTION
**Pull request by rgbcolor**

### CODE REVIEW

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+title: "How to have a side project"
+social: true
+author: Maria Matveeva
+twitter: "rgbcolor"
+summary: "Focus on one project at a time. Bonus: see what DockYarders do outside work!"
+published: true
+tags: art, design, creativity, culture, inspiration, team
+---
+
+Pretty much every designer I know has a side project (or five!) They are often in a field related to design and art, but do not replicate exactly what we do at our full time job. Side projects and hobbies are great for relaxing, while adding extra skills or addressing a passion that does not yet have a place in the career.
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: five!). They

Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: what *they* do at

Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: in *one's* career

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+Here’s what I learned about what works well, and what can be problematic for side projects I’ve had over the years. Hope it helps you in your own creative pursuit!
+
+![watercolor sketch with abstract shapes](https://i.imgur.com/28d3H8T.jpg)
+*My watercolor sketch to study shapes and colors from Wassily Kandinsky*
+
+
+## 1: Different enough
+To let your brain rest and relax, a side project should be different enough from what you do during the day. When a day’s work is mostly at the computer, I feel a need to switch to analog afterwards, and still have lots of energy. If this is a [design sprint](https://dockyard.com/design-sprints) week, and I’m on my feet most of the day, sketching or writing on the whiteboard - then curling up on my couch with a laptop feels like a welcome break. Essentially, I have different types of energy I could use up during the day, and switching between activities allows me to get more done.
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Completely agree with this!

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+Another principle to keep in mind is to avoid conflict of interest between work and personal projects. When they are different enough, there is a clear separation between the two. I won’t resent filling my week-ends with the personal project, because it does not feel like work. It feels like play!
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Either:
avoid *a* conflict
or
avoid *conflicting interests* between

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+
+## Focus
+
+I now know I can’t get good results from trying to juggle several side projects at a time. But deciding to do just one thing can feel scary. Will I ever get to work on my other things? But it’s the only sure way to make progress.
+
+![Color blocks and sticky notes](https://i.imgur.com/Lveuv3z.jpg)
+*Johannes Itten’s classic Elements of Color (which in forms both art and design) alongside some sticky notes from my UX design life*.
+
+In 2015, I decided to focus on one side project: my exploration of the [overlap between design and modern art](http://www.designartpractice.com/). I put away my sewing machines, and set my Easy shop in holiday mode. I even stopped nagging myself to keep practicing with watercolors or hand lettering. I now have a more realistic to-do list, and can feel “done” and accomplished with the weekly illustrated articles that I plan to write for a year.
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: I have no idea what "Easy shop" is :-(

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+I went around and asked my intimidatingly talented colleagues to share examples of what they’re up to. Here they go!
+
+
+## Estelle
+
+![pencil portrait by Estelle](https://i.imgur.com/4Si2OgA.jpg)
+*Pencil Portrait from my very limited [portfolio](https://www.behance.net/edeblois)*
+
+**What do you do outside work?**
+I tend to go back to the things I enjoyed doing before discovering the joy of programming. I have a variety of hobbies that I try to juggle as time permits, ranging from drawing or painting, practicing photography (I took an online photography course at the Photography Institute a few years ago and try to keep), practicing piano (I’ve been taking weekly piano lessons for the past six years), and just gaming.
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Looks like this sentence got cut off:

"and try to keep..."

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+*2015 Pukwudgie Time Trial - Assonet, MA*
+
+**What do you do outside work**?
+Competitive mountain biking
+
+**How do you feel it feeds into your work, if at all**?
+Over the years I have participated in many team sports where the performance of the collective group dictates outcome of the competition.  Competitive mountain biking has taken out the variable of team and placed full responsibility on me.  Any success I have realized in this endeavor (which has been minimal to date) is a fringe benefit of deliberate training, sacrifice, failure and exhibition of mental toughness.  
+
+I love the fact teams are a significant part of my professional life - It is the only way I would want it to be.   However, outside of work I have chosen something that is “Different Enough” to keep my other lesser used mental and physical muscles engaged.
+## Lauren 🐷 paints!
```
Comment (michaeldupuisjr) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Need a line break above this line.

File: source/posts/2016-02-10-how-to-have-a-side-project.md (hunk 5 above)
Comment (rgbcolor) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Good catch @michaeldupuisjr ! @brzpegasus ^ could you edit the sentence? :)

Comment (brzpegasus) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: Oh yeah, it's supposed to say "keep up".

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+title: "How to have a side project"
+social: true
+author: Maria Matveeva
+twitter: "rgbcolor"
+summary: "Focus on one project at a time. Bonus: see what DockYarders do outside work!"
+published: true
+tags: art, design, creativity, culture, inspiration, team
+---
+
+Pretty much every designer I know has a side project (or five). They are often in a field related to design, but do not replicate exactly what what they do at their full time job. Side projects and hobbies are great for relaxing, while adding extra skills or addressing a passion that does not yet have a place in one's career.
```
Comment (brzpegasus) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: extra "what" in "replicate exactly what what they do"

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+
+I eventually climbed out of this. I negotiated Fridays off at work, trained the designer who would replace me, put together a portfolio and moved to Canada for a year to study. The experience left me with a better understanding of how important focus really is.
+
+
+## Focus
+
+I now know I can’t get good results from trying to juggle several side projects at a time. But deciding to do just one thing can feel scary. I often wonder if I'll ever get to work on my other projects. But it’s the only sure way to make progress.
+
+![Color blocks and sticky notes](https://i.imgur.com/kIUHUpm.jpg)
+*Johannes Itten’s classic Elements of Color (which in forms both art and design) alongside some sticky notes from my UX design life*.
```
Comment (brzpegasus) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: "which in forms both art and design" -- is there a word missing, or extra word that shouldn't be there?

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+
+In 2015, I decided to focus on one side project: my exploration of the [overlap between design and modern art](http://www.designartpractice.com/). I put away my sewing machines, and set my Etsy shop in holiday mode. I even stopped nagging myself to keep practicing with watercolors or hand lettering. I now have a more realistic to-do list, and can feel “done” and accomplished with the weekly illustrated articles that I plan to write for a year.
+I’ll be honest: I miss the other side projects I had put on hold. But am OK with not doing them for a while so that once I get to them, I’ll be able to make progress and commit to giving those other passions the focused time they deserve.
```
Comment (brzpegasus) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: "But I am OK"

File: source/posts/2016-02-10-how-to-have-a-side-project.md [hunk 10]
```diff
@@ -0,0 +1,244 @@
...
+My after-hours time is spent on open source software. Right now I am working on Elixir tools that will be used to support our future application development efforts.
+
+
+## Steve
+
+![Black and white photo of hands](https://i.imgur.com/Smaad8H.jpg)
+*A self portrait done on 35mm black and white film*.
+
+![abstract photo of ice](https://i.imgur.com/odVVRMv.jpg)
+*This is a picture I took of ice. It’s appearance is more smokey than usual due to melting and refreezing, and because of the size of the river I took this in, I was able to capture somewhere around 15 original compositions*.
```
Comment (brzpegasus) [inline source/posts/2016-02-10-how-to-have-a-side-project.md]: "It's" -> "Its"

**Pull request merged**